src/o365/flow/commands/flow-remove.ts

```diff
@@ -0,0 +1,171 @@
+import commands from '../commands';
+import GlobalOptions from '../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../Command';
+import request from '../../../request';
+import AzmgmtCommand from '../../base/AzmgmtCommand';
+import * as os from 'os';
```

Comment:
We're not using `os` in this command and should remove the reference

Reply:
It's used on line 151 in the example remarks, as per previous comment, I just took from other examples of flow commands which contain the same use of `${os.EOL}`.